.github/workflows/email.yml

```diff
@@ -3,21 +3,25 @@ name: email
 on:
   push:
     branches:
-      - main 
+     # - main 
       
 jobs:
   
   
   send_email:
    # run this workflow step only when the PR is merged 
-    if:  github.event.pull_request.merged == true 
+    #if:  github.event.pull_request.merged == false 
```

Comment:
Commented out to see if this is what is causing the email step to be skipped. 
Will remove this line if this dix works in a followup review.